types/node/util.d.ts

```diff
@@ -1173,6 +1173,45 @@ declare module "util" {
          */
         const custom: unique symbol;
     }
+    /**
+     * Stability: 1.1 - Active development
+     * Given an example `.env` file:
+     *
+     * ```js
+     * const { parseEnv } = require('node:util');
+     *
+     * parseEnv('HELLO=world\nHELLO=oh my\n');
+     * // Returns: { HELLO: 'oh my' }
+     * ```
+     * @param content The raw contents of a `.env` file.
+     * @since v20.12.0
+     */
+    export function parseEnv(content: string): object;
+    /**
+     * Stability: 1.1 - Active development
+     *
+     * This function returns a formatted text considering the `format` passed.
+     *
+     * ```js
+     * const { styleText } = require('node:util');
+     * const errorMessage = styleText('red', 'Error! Error!');
+     * console.log(errorMessage);
+     * ```
+     *
+     * `util.inspect.colors` also provides text formats such as `italic`, and `underline` and you can combine both:
+     *
+     * ```js
+     * console.log(
+     *   util.styleText('underline', util.styleText('italic', 'My italic underlined message')),
+     * );
+     * ```
+     *
+     * The full list of formats can be found in [modifiers](https://nodejs.org/docs/latest-v20.x/api/util.html#modifiers).
+     * @param format A text format defined in `util.inspect.colors`.
+     * @param text The text to to be formatted.
+     * @since v20.12.0
+     */
+    export function styleText(format: string, text: string): string;
```

Comment:
Thank you for adding this!

Considering the `format` parameter can only accept a single option at a time, it would be helpful to use a union type to limit the options to only those the code will use instead of `string`:
```suggestion
    // https://nodejs.org/docs/latest/api/util.html#foreground-colors
    type ForegroundColors =
      | 'black'
      | 'blackBright'
      | 'blue'
      | 'blueBright'
      | 'cyan'
      | 'cyanBright'
      | 'gray'
      | 'green'
      | 'greenBright'
      | 'grey'
      | 'magenta'
      | 'magentaBright'
      | 'red'
      | 'redBright'
      | 'white'
      | 'whiteBright'
      | 'yellow'
      | 'yellowBright';

    // https://nodejs.org/docs/latest/api/util.html#background-colors
    type BackgroundColors =
      | 'bgBlack'
      | 'bgBlackBright'
      | 'bgBlue'
      | 'bgBlueBright'
      | 'bgCyan'
      | 'bgCyanBright'
      | 'bgGray'
      | 'bgGreen'
      | 'bgGreenBright'
      | 'bgGrey'
      | 'bgMagenta'
      | 'bgMagentaBright'
      | 'bgRed'
      | 'bgRedBright'
      | 'bgWhite'
      | 'bgWhiteBright'
      | 'bgYellow'
      | 'bgYellowBright';

    // https://nodejs.org/docs/latest/api/util.html#modifiers
    type Modifiers =
      | 'blink'
      | 'bold'
      | 'dim'
      | 'doubleunderline'
      | 'framed'
      | 'hidden'
      | 'inverse'
      | 'italic'
      | 'overlined'
      | 'reset'
      | 'strikethrough'
      | 'underline';

    function styleText(format: ForegroundColors | BackgroundColors | Modifiers, text: string): string;
```